.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     'react/no-unknown-property': 0,
     'react/prop-types': 0,
     'react/react-in-jsx-scope': 0,
+    'react-hooks/exhaustive-deps': 0,
```

Comment:
Please remove this item. it is necessary for development. We have to see what dependencies are missing.